File: server/src/main/java/model/Status.java
package model;

import com.influxdb.annotations.Column;
import com.influxdb.annotations.Measurement;

import java.time.Instant;

@Measurement(name = "status")
public class Status {

    @Column(tag = true)
    public int id;

    @Column
    public String value;

    @Column(timestamp = true)
    public Instant instant;
}

File: client/lib/status.dart

class Status {
  final String? id;
  final String? position;
  final String? instant;

  Status({
    required this.id,
    required this.position,
    required this.instant
  });

  @override
  String toString() {
    return "$id;$position;$instant";
  }
}